alf/environments/utils.py

```diff
@@ -172,6 +171,12 @@ def create_environment(env_name='CartPole-v0',
 
     """
 
+    if env_name is None:
+        env_name = 'CartPole-v0'
```

Comment:
This seems to be the same as using ``CartPole-v0`` as the default value for the argument ``env_name``?